src/components/Login.tsx

```diff
@@ -79,7 +79,13 @@ const Login = () => {
     }, [token, login, logout])
 
     return (
-        <div>
+        <div
+            style={{
+                position: 'absolute',
+                left: 0,
+                background: '#4444FF55',
+                zIndex: 50,
+            }}>
```

Comment:
pls consider using a styled component here

Reply:
common, it's a dumb component that will be rewritten